src/__tests__/index.test.js

```diff
@@ -0,0 +1,20 @@
+import renderer from 'react-test-renderer';
```

Comment:
**I looked through your app, it's perfect!**

- [ ] However, when it comes to testing. I only can see snapshots of tests of your components. Students are encouraged to write tests to test the functionality of your app for example fetching and filtering... with a one-word integration test. Kindly add integration tests to your app, Thank you!